distribution/docs/src/main/resources/content/_configuring/global-settings.adoc

```diff
@@ -477,6 +477,13 @@ Thumbs.db
 |zookeeperhost1:2181,zookeeperhost2:2181,zookeeperhost3:2181
 |No
 
+|Solr max heap size
+|solr.max.heap.size
+|String
+|Memory settings for the Solr server
```

Comment:
❓ How about _Memory allocated to the Solr Java process_ or _Memory allocated to the Solr server_? Also, should we mention that this only applies to non-SolrCloud installations? Might actually be something we want to mention for all the properties, or group them by configuration type maybe?

Reply:
1. I'm going with _Memory allocated to the Solr Java process_ 
2. I think it should be mentioned. I'll look around at how to do it. 

I have never configured Solr Cloud myself. It looks like the only two properties Solr Cloud Provider needs are the name of the client and the zookeeper  hosts/ports. Is that correct?

Reply:
That's correct. DDF needs the `solr.client` property and the Solr client needs the list of zookeeper nodes.

Reply:
@pklinef, @rzwiefel, does SolrCloud need `solr.data.dir` as well (https://github.com/codice/ddf/blob/master/platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/SolrCloudClientFactory.java#L176) or is it an optional property?

Reply:
@lessarderic I don't believe SolrCloud needs it - it is only for dynamic core creation with non-solrcloud since it doesn't support the same capabilities as SolrCloud does.  So its required for non-SolrCloud installations but optional/not applicable for SolrCloud.